terratest/utils/extensions.go

```diff
@@ -271,6 +271,19 @@ func (i *Instance) GetLocalTargets() []string {
 	return dig
 }
 
+func (i *Instance) GetInternalNodeIPs() (result []string) {
+	result = make([]string,0)
+	nodes := k8s.GetNodes(i.w.t,i.w.k8sOptions)
+	for _,n := range nodes {
+		for _, a := range n.Status.Addresses {
+			if a.Type == corev1.NodeInternalIP {
```

Comment:
@k0da, @ytsarev - sorry, the problem is this line ?

Reply:
@kuritka for me it is the overall expectation of https://github.com/k8gb-io/k8gb/pull/534/files#diff-02a37d22e625bb75658c30a4e16e1d0408225b83209862a48285b63c4af98f35R37-R38